infra/experimental/SystemSan/SystemSan.cpp

```diff
@@ -488,5 +521,6 @@ int main(int argc, char **argv) {
   g_root_pid = pid;
   std::map<pid_t, Tracee> pids;
   pids.emplace(pid, Tracee(pid));
+  root_pids.emplace(pid, ThreadParent(pid));
```

Comment:
This is a bit confusing: the `ThreadParent` for `pid` is `pid`?  What does that mean?

Reply:
Actually disregard, I misread this.